pennylane/ops/qubit/state_preparation.py

```diff
@@ -89,3 +92,33 @@ def decomposition(state, wires):
 
     def adjoint(self):
         raise qml.ops.AdjointError("No adjoint exists for QubitStateVector operations.")
+
+
+class QubitDensityMatrix(Operation):
+    r"""QubitDensityMatrix(state, wires)
+    Prepare subsystems using the given density matrix.
+    If not all the wires are specified, remaining dimension is filled by |0><0|,
+    where leftmost axes has 1 on its element when transposed as |0><0| ⊗ ρ.
```

Comment:
There seems to be a problem building the docs, try this: 
```suggestion
    If not all the wires are specified, remaining dimension is filled by :math:`\vert 0 > < 0 \vert`,
    where leftmost axes has 1 on its element when transposed as :math:`\vert 0 > < 0 \vert \otimes \rho`.
```